ServerCore/Pages/Shared/_Layout.cshtml

```diff
@@ -66,11 +66,25 @@
                    render-mode="ServerPrerendered" />
     }
 
-    <!-- TODO: Make this actually scalable and correct -->
     @{
+        // Keep the old path for events prior to 2024 so they aren't broken
+        // For all future events, resources global to all pages for an event should be uploaded to the shared event resources page
+        // Note that the link for development may be different on your machine
         if ((ev?.HomePartial != null) && ((ev.HomePartial.StartsWith("pd")) || (ev.HomePartial.StartsWith("nipd"))))
         {
-            <link rel="stylesheet" href="~/css/@(ev.UrlString)/styles.min.css" />
+            if (ev.EventBegin.Year < 2024)
+            {
+                <link rel="stylesheet" href="~/css/@(ev.HomePartial)/styles.min.css" />
+            }
+            else
+            {
+                <environment include="Development">

```

Comment:
Same FileManager trick here

Reply:
Also, I assume we want this stylesheet set up even if the event does not begin with pd or nipd...

Reply:
To the second question, no, this was the original logic if you look through the file history.  We've only ever applied styles like this to puzzleday and only have style files in wwwroot for pd and nipd, so that is technically not a change in this PR.